pkgs/development/haskell-modules/configuration-ghc-8.0.x.nix

```diff
@@ -53,4 +53,22 @@ self: super: {
     license = pkgs.stdenv.lib.licenses.bsd3;
   }) {};
 
+  # Remove upper limit of dependency on time < 1.6
+  # Fix overlapping instance in tests.
+  libmpd = overrideCabal super.libmpd (drv: {
+     preConfigure = ''
+        sed -i -e 's,time .* < *1.6,time >= 1.5,' libmpd.cabal
+        sed -i -e '54s/instance /instance {-# OVERLAPS #-}/' tests/Arbitrary.hs
+     '';
+  });
+
+  xmobar = (overrideCabal super.xmobar (drv: {
+    # Skip -fwith_datezone
+    configureFlags = [ "-fwith_xft" "-fwith_utf8" "-fwith_inotify"
```

Comment:
I'm pretty sure that those changes are unnecessary, i.e. they have been applied in `master` already.